src/gsi/chemmod.f90

```diff
@@ -90,7 +93,9 @@ module chemmod
 
   logical :: aero_ratios
 
-  logical :: oneobtest_chem,diag_incr,berror_chem,berror_fv3_cmaq_regional
+  logical :: oneobtest_chem,diag_incr,berror_chem
+  logical :: berror_fv3_cmaq_regional,berror_fv3_sd_regional
```

Comment:
I searched this PR, it looks like we don't need separate berror_fv3_sd_regional and berror_fv3_cmaq_regional?

berror_fv3_sd_regional is only used at Line 460 of src/gsi/m_berror_stats_reg.f90 and there is no difference if we use berror_fv3_cmaq_regional instead. Is it possible to just use berror_fv3_cmaq_regional, or combine them to one logical variable, such as berror_fv3_aero_regional?

Reply:
The current ones make it more clear to know which B file will be used, and the previous retros runs can repeat without any changes of GSI namelist parameter. This is good to users. 
Your suggestion will make the changes be general and good in code development. 
So what do you think?

Reply:
Do you plan to use different BEC for cmaq and sd? Or will they be the same as they are right now?

Reply:
Got it. Thanks for the explanation!